app/utils/sharetribe_logger.rb

```diff
@@ -0,0 +1,55 @@
+class SharetribeLogger
+  attr_writer(
+    :community_id,
+    :community_ident,
```

Comment:
Should we call these marketplace_id and marketplace_ident as those are the new terms even though the data model under the hood still talks about communities?